packages/wallet/backend/src/account/service.ts

```diff
@@ -300,16 +300,17 @@ export class AccountService implements IAccountService {
   }
 
   public async createDefaultAccount(
-    userId: string
+    userId: string,
+    isBoutique: boolean = false
```

Comment:
change `isBoutique` param to `name` with default value 'USD Account'
this why the method can  be used in the future for other type of accounts without any changes